modules/styling/stories/mdx/Overview.mdx

```diff
@@ -1,138 +1,235 @@
+
+These packages work together to provide a CSS-in-JS experience during development while enabling optimized static CSS in production.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-styling
 ```
 
-For further information, please read our GitHub discussion on
-[the future of styling](https://github.com/Workday/canvas-kit/discussions/2265)
+## Usage
+
+```tsx
+import React from 'react';
+import {createRoot} from 'react-dom/client';
 
-## What is Canvas Kit Styling?
+import {createStyles} from '@workday/canvas-kit-styling';
 
-Canvas Kit Styling includes two things:
+const myStyles = createStyles({
+  backgroundColor: 'red',
+}); // returns the CSS class name created for this style
+
+myStyles; // something like "css-{hash}"
 
-1. A utility function wrapper around `@emotion/css`
-2. An optional static compiler to remove most of the runtime
+const domNode = document.getElementById('root');
+const root = createRoot(domNode);
 
-### Utility Functions
+root.render(<div className={myStyles}>Hello!</div>);
+```
 
-This packages provides three utility functions to make it easier to style element-based components.
-The following is a brief description of each function. If you'd like to read a more in-depth
-discussion of each, our [API docs](/docs/features-styling-api--create-styles).
+## Development
+
+Canvas Kit Styling comes with a runtime that doesn't need anything special for developement. The
+runtime uses `@emotion/css` to include your styles on the page.
+
+## Production
+
+If you wish to use the static compilation, you must use the `@workday/canvas-kit-styling-transform`
+package. Add the following to your project's `tsconfig.json` file:
+
+```json
+{
+  "compilerOptions": {
+    // other options
+    "plugins": [
+      {
+        "transform": "@workday/canvas-kit-styling-transform",
+        "prefix": "css",
+        "fallbackFiles": [""]
+      }
+    ]
+  }
+}
+```
+
+This adds a list of plugins to use when transforming TypeScript files into JavaScript files. The
+[ts-patch](https://www.npmjs.com/package/ts-patch) projects uses the `plugins` when running
+transforms.
+
+### Webpack
+
+You will need to transform TypeScript files using the `ts-patch` which is the same as the TypeScript
+tanspiler except it uses TypeScript's
+[transform API](https://levelup.gitconnected.com/writing-a-custom-typescript-ast-transformer-731e2b0b66e6)
+to transform code during compilation.
+
+In your webpack config, you add the following:
+
+```js
+{
+  rules: [
+    //...
+    {
+      test: /.\.tsx?$/,
+      loaders: [
+        // ts-loader
+        {
+          loader: require.resolve('ts-loader'),
+          options: {
+            compiler: 'ts-patch/compiler',
+          },
+        },
+        // OR awesome-typescript-loader
+        {
+          loader: require.resolve('awesome-typescript-loader'),
+        },
+      ],
+    },
+  ];
+}
+```
+
+## Core Styling Approaches for Static Styling
```

Comment:
add a small paragraph here summarizing how many approaches we have. this would help before just jumping right into the approaches